### PR TITLE
Add Elasticsearch Docker Image

### PR DESCRIPTION
**Pull request by tarrow**

### CODE REVIEW

File: Docker/build/Elasticsearch/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ARG ELASTICSEARCH_VERSION
+FROM elasticsearch:${ELASTICSEARCH_VERSION}
+RUN ./bin/elasticsearch-plugin install org.wikimedia.search:extra:6.5.4
```
Comment (mariushoch) [inline Docker/build/Elasticsearch/Dockerfile:3]: Should this also use ${ELASTICSEARCH_VERSION}?

File: Docker/build/Elasticsearch/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+RUN ./bin/elasticsearch-plugin install org.wikimedia.search.highlighter:experimental-highlighter-elasticsearch-plugin:6.5.4
```
Comment (mariushoch) [inline Docker/build/Elasticsearch/Dockerfile:4]: Should this also use ${ELASTICSEARCH_VERSION}?

File: Docker/build/Elasticsearch/Dockerfile (hunk 1 above)
Comment (tarrow) [reply Docker/build/Elasticsearch/Dockerfile:3]: done!

File: Docker/build/Elasticsearch/Dockerfile (hunk 2 above)
Comment (tarrow) [reply Docker/build/Elasticsearch/Dockerfile:4]: done!

File: versions/REL1_34.env [hunk 3]
```diff
@@ -7,4 +7,6 @@ MEDIAWIKI_MAJOR_VERSION=1.34
 MEDIAWIKI_VERSION=1.34.0
 
 RELEASE_MAJOR_VERSION=1.34
-RELEASE_VERSION=1.34.0
\ No newline at end of file
+RELEASE_VERSION=1.34.0
+
+ELASTICSEARCH_VERSION=6.5.4
```
[the remark this answers is not in the document]
Comment (tarrow) [reply versions/REL1_34.env:12]: Which `.env`? It is already in the root one I think?

[the remark this answers is not in the document]
Comment (toban) [reply versions/REL1_34.env:12]: you are correct! This comment looks like a brain-fart.

**Pull request merged**